src/components/ThreeDEditor.jsx

```diff
@@ -95,8 +95,8 @@ export class ThreeDEditor extends React.Component {
         const { material } = nextProps;
         if (material) {
             this.setState({
-                originalMaterial: material,
-                // material: material.clone(),
+                material,
+                originalMaterial: material.clone(),
```

Comment:
The logic seems to have changed, `.clone()` is not always producing equivalent to original